drivers/hmis/app/graphql/types/hmis_schema/ce_referral.rb

```diff
@@ -23,6 +23,7 @@ class HmisSchema::CeReferral < Types::BaseObject
     field :target_enrollment, Types::HmisSchema::Enrollment, null: true
     field :swimlanes, [HmisSchema::CeReferralSwimlane], null: false
     field :workflow_template_name, String, null: true
+    field :events, HmisSchema::CeReferralEvent.page_type, null: false, method: :timeline_events
```

Comment:
I think we might get confused later on when we start to resolve CE Events on referrals as well. I'd suggest calling these "audit events  for example:
```
referral {
  ceEvents { .. } # this will probably be added later. Types::HmisSchema::Event
  auditEvents { .. } # Types::HmisSchema::CeReferralAuditEvent
}
```